target/linux/octeon/image/Makefile

```diff
@@ -23,22 +23,24 @@ define Build/strip-kernel
 endef
 
 define Device/generic
+  DEVICE_MODEL := Generic
```

Comment:
Should there be a vendor set?

Reply:
Just keep DEVICE_TITLES for Generic builds which don't belong to any specific board.